endpoints/remotehost/remotehost

```diff
@@ -199,17 +201,13 @@ function cleanup_osruntime() {
 
         if [ -e $endpoint_run_dir/chroot-container-mount.txt ]; then
             container_mount=`cat $endpoint_run_dir/chroot-container-mount.txt`
-            for fs in proc dev dev/hugepages proc sys lib/modules usr/src boot; do
-                if [ -e $container_mount/$fs ]; then
```

Comment:
This line was broken since it's doing the path check on the controller instead of one the remotehost.  I attempted to fix it but had other issues that made me just ignore this and get rid of this check...